packages/perseus-editor/src/diffs/answer-area-diff.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from "react";
+import _ from "underscore";
+
+import DiffEntry from "./shared/diff-entry";
+import performDiff from "./shared/widget-diff-performer";
+
+import type {PerseusAnswerArea} from "@khanacademy/perseus-core";
+
+type AnswerAreaDiffProps = {
+    after: PerseusAnswerArea;
+    before: PerseusAnswerArea;
+    title: string;
+};
+
+export class AnswerAreaDiff extends React.Component<AnswerAreaDiffProps> {
+    render(): React.ReactNode {
+        const {after, before, title} = this.props;
+        const diff = performDiff(before, after);
+        return (
+            <div>
```

Comment:
Somewhat trivial question: Does this need to be wrapped in a `div`? Or can we just make it a fragment `<>` so that we'd have to look through fewer layers in the DOM when doing any future investigations with browser tools.